airflow/ui/src/pages/Variables/VariableAction.tsx

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+// TODO: Disable lint will be removed once the implementation completed
+
+/* eslint-disable no-alert */
+import { IconButton } from "@chakra-ui/react";
+import { FiEdit, FiEye, FiMoreVertical, FiTrash } from "react-icons/fi";
+
+import { Menu } from "../../components/ui/Menu";
+
+type VariableActionsProps = {
+  readonly variableKey: string;
+};
+
+export const VariableActions = ({ variableKey }: VariableActionsProps) => (
+  <Menu.Root>
+    <Menu.Trigger asChild>
+      <IconButton aria-label="Action menu" variant="ghost">
+        <FiMoreVertical />
+      </IconButton>
+    </Menu.Trigger>
+
+    <Menu.Content>
+      <Menu.Item
+        onClick={() =>
+          alert(`To be implemented: Selected key is ${variableKey}`)
```

Comment:
To be consistant with other `Clear` and `Trigger` Actions, I would recommend not putting that under a menu but side by side directly in the table cell. Also the action needs a tooltip and there is already a component that can help achieving that:
 `ActionButton`. You can also check `ClearRunButton` and `TriggerDagButton` that leverage this `ActionButton` component.

Reply:
> Because view would only make sense if we have a truncate on the display, |

I'm not sure I understand that. What are you referring to by `view` in this context ?

Reply:
Ohh okay.

So view is also an action component which the legacy UI had it on the variables list page, where when user clicked it, they get to see the details, such as key, values and description. 

Now columns in the legacy ui had a fix width, so any of the content overflowing was truncated and ended with three dots (...) , so to see overall content user had to use the view action button.

Now in the new UI, tables have horizontal scrolling available. So all the data gets printed on the list page itself, since no width restrictions. In my opinion this will be a problem, since keys supports 250 chars and other two parameter value and description supports more than key also. This will not allow page to look good. So I think we need restrict the width, block horizontal scrolling and have a view button. What would you say?

Reply:
Yes I also think that displaying everything in the table (very long description or key) will be a problem and make the table look bad.

I would be in favour of truncating / ellipsis too long string in the table display. And either show the full text in a “tooltip” or add a “detail” view onClick that will get to a dedicated page with full text displayed for those values.

Though I don’t think we need an “action” “view”. A simple link on the key as we do for tasks or dagrun is enough.

Reply:
Ohh okay!

Thanks, In that case I think we can merge this PR, that issues can be addressed in different PR since it's not relevant to this.